.github/workflows/pr-builder.yml

```diff
@@ -19,7 +19,7 @@ jobs:
 
     strategy:
       matrix:
-        java-version: [ 11, 17 ]
+        java-version: [ 11.0.19+7 ]
```

Comment:
any reason for removing java version 17?

Reply:
I followed this https://github.com/wso2-extensions/identity-inbound-auth-oauth/blob/6cb4c30737e62537aad1d94c07e88283dcf80198/.github/workflows/pr-builder.yml#L23 as a workaround for the build failure.

Reply:
include [jdk-17.0.7+7](https://github.com/adoptium/temurin17-binaries/releases/tag/jdk-17.0.7%2B7) as well

Reply:
Added in [bea2a6a](https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/478/commits/bea2a6a4c22c549bbf6db771c25d1df238cd854a)